src/Workspaces/Core/Portable/Workspace/Solution/ProjectInfo.cs

```diff
@@ -82,6 +85,11 @@ public sealed class ProjectInfo
         /// </remarks>
         internal string? DefaultNamespace => Attributes.DefaultNamespace;
 
+        /// <summary>
+        /// Algorithm to calculate content checksum for debugging purposes.
+        /// </summary>
+        internal SourceHashAlgorithm ChecksumAlgorithm => Attributes.ChecksumAlgorithm;
```

Comment:
Any reason not to call the member SourceHashAlgorithm? Otherwise this type has both a Checksum property and a ChecksumAlgorithm property and those have no relation to each other.

Reply:
The Checksum property is actually `IChecksummedObject.Checksum`.